spark-tpcds-benchmark-runner/build.gradle

```diff
@@ -24,12 +24,16 @@ dependencies {
     implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8'
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-guava'
+    implementation 'com.github.rholder:guava-retrying'
     implementation 'com.google.guava:guava'
     implementation 'com.palantir.common:streams'
     implementation 'com.palantir.safe-logging:preconditions'
     implementation 'com.palantir.conjure.java.runtime:client-config'
     implementation 'org.apache.spark:spark-sql_2.12'
     implementation 'org.apache.spark:spark-hadoop-cloud_2.12'
+    implementation('org.mapdb:mapdb') {
+        exclude group: 'net.jpountz.lz4', module: 'lz4'
```

Comment:
curious, why is this necessary?

Reply:
this library changed its maven coordinate to `org.lz4:lz4-java` in a later release, which spark depends on; having both on the classpath caused problems. See https://github.com/palantir/spark-tpcds-benchmark/blob/e49806fd0ee5cf0e93b15d64d386d5d3c5b8f470/versions.lock#L213